src/patternfly/layouts/Toolbar/styles.scss

```diff
@@ -34,14 +62,23 @@
 }
 
 // Group
-.pf-l-toolbar__group:not(:last-child) {
+.pf-l-toolbar__group {
+  // adds top margin to all group divs
+  margin-top: var(--pf-l-toolbar__group--MarginTop);
   margin-right: var(--pf-l-toolbar__group--MarginRight);
+
+  .pf-l-toolbar__item {
+    margin-top: var(--pf-l-toolbar__group-item--MarginRight-Zero);
+    margin-right: var(--pf-l-toolbar__group-item--MarginRight-Zero);
+  }
+
+  .pf-l-toolbar__item:last-child {
+    margin-right: var(--pf-l-toolbar__group-item--MarginRight-Zero);
```

Comment:
var should be `--pf-l-toolbar__group-item--last-child--MarginRight`

Reply:
this selector isn't necessary though because you are already setting `margin-right` to "0" in the block above it. However, these styles only apply to items that are in a group, so there is still a right margin on the last child of items that are the direct children of a toolbar.